linux-package.sh

```diff
@@ -16,15 +16,14 @@ javapackager -deploy \
     -BappVersion=$ver \
     -Bcategory=Office,Finance \
     -BlicenseType=Apache \
+    -BlicenseFile=LICENSE \
     -Bemail=contact@vinumeris.com \
     -Bicon=client/icons/icon.png \
     -native deb \
     -name lighthouse \
-    -title Lighthouse \
+    -title "Lighthouse: a peer to peer crowdfunding app that uses Bitcoin" \
     -vendor Vinumeris \
     -outdir deploy \
     -appclass lighthouse.Main \
-    -srcfiles $build \
+    -srcdir updates/builds/processed \
```

Comment:
Ah, so that's why my attempts to make LICENSE work failed - it has to be listed in the source files. 

I've been meaning to improve the UpdateFX workflow I use as the current one is prone to mistakes. Your approach of emptying the processed directory might work, I need to check that the UpdateFX tool is OK with that. I'll get back to you soon. The current commit looks OK assuming it doesn't mess with the updates system. Thanks for fixing this.